files/generate-vars

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/python
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
+import os.path
+import shlex
+import subprocess
+import sys
+
+from subprocess import call
+
+
+class GenerateMappingFile():
+    prefix = "[Generate Mapping File] "
+    ansible_play = "../examples/dr_generate_var.yml"
+    dr_tag = "generate_mapping"
+    dr_conf_file = "dr.conf"
+
+    def run(self):
+        print("\n%sStart Generate Variable Mapping File "
+              "For oVirt Ansible Disaster Recovery" % self.prefix)
+        site, username, password, ca, output_file = self._init_vars()
+        print("\n%ssite address: %s \n"
+              "%susername: %s \n"
+              "%spassword: \n"
```

Comment:
Maybe: ****** instead of empty would be nicer..

Reply:
indeed, done